### ISSUE TITLE
Integrate with Sublime Merge?

### ISSUE THREAD

**Issue by astronouth7303** [flathub/com.sublimetext.three#19]
Sublime Text has features/hooks to open Sublime Merge.

Currently, this doesn't seem to work, even if both paks are installed.

**Comment by astronouth7303** [comment]
Yes, I'm aware there's probably limitations in the flatpak specs about this.

Suggestions:
* Get the specs changed
* Publish a Sublime Suite pak

**Comment by ptomato** [comment]
Thanks for the suggestion, and sorry for the delay in replying!

Someone actually already built in an extension point to the Sublime Text flatpak for the purpose of supporting Sublime Merge: https://github.com/flathub/com.sublimetext.three/blob/master/com.sublimetext.three.yaml#L23-L25
It just needs someone to hook it up. I don't really have time to do this right now, and neither do I know how off the top of my head, but I will gladly review and merge a pull request that does this.
